### PR TITLE
handle `.exactly(0)`

### PR DESCRIPTION
**Pull request by n3dst4**
This is a minor item, but I noticed that if I had a test use the chain `expect(wrapper).to.have.exactly(0).descendants("foo")` it was being treated as if it was `expect(wrapper).to.have.descendants("foo")`.

Obviously the workaround is to use `.to.not.have.descendants(...)` but I figured it was worth fixing.

### CODE REVIEW

File: src/assertions/descendants.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 export default function descendants ({ wrapper, markup, arg1, sig, flag }) {
   const exactlyCount = flag(this, 'exactlyCount')
 
-  if (exactlyCount) {
```
Comment (ljharb) [inline src/assertions/descendants.js]: What about null or false?

This should probably be `exactlyCount || exactlyCount === 0`.

Also, it should probably throw on a non-number input, but that can be a separate PR.

File: test/exactly.test.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ class Fixture extends React.Component {
 
 const it = createTest(<Fixture />)
 
-describe('#exactly', () => {
+describe.only('#exactly', () => {
```
Comment (ljharb) [inline test/exactly.test.js]: Please revert this - also, you can use `--grep` in testing and you'll never need to manually mark a test as .only again :-)

File: src/assertions/descendants.js (hunk 1 above)
Comment (n3dst4) [inline src/assertions/descendants.js]: Yeah.
